web/beacon-app/src/features/projects/types/newProjectFormService.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable prettier/prettier */
+import { t } from '@lingui/macro';
+import { useFormik } from 'formik';
+import { object, string } from 'yup';
+
+import { NewProjectDTO } from './createProjectService';
+
+export const FORM_INITIAL_VALUES = {
+  name: '',
+  description: '',
+} satisfies Omit<NewProjectDTO, 'tenantID'>;
+
+export const FORM_VALIDATION_SCHEMA = object({
```

Comment:
In SC-15723 constraints for the project name and description are listed. Could you update the validation schema to display an error if the project name is more than 512 characters? Also, could it display an error if the description is more than 2,000 characters, if one is provided?

Reply:
Good catch. The better way to handle that is to set the max length to the textarea component. i will also add a length flag to let user know the remaining character.